src/sql/workbench/services/connection/browser/connectionBrowseTab.ts

```diff
@@ -150,9 +150,17 @@ export class ConnectionBrowserView extends Disposable implements IPanelView {
 	renderTree(container: HTMLElement): void {
 		this.treeContainer = container.appendChild(DOM.$('div'));
 		this.treeLabels = this._register(this.instantiationService.createInstance(ResourceLabels, this));
+		this.treeMenus = this.instantiationService.createInstance(TreeMenus);
+		this.treeMenus.contextKeyService = this.contextKeyService;
```

Comment:
Why not just have the service passed in via the instantiation service? Or at the very least as an actual parameter to the constructor.

Reply:
😄 I don't know what I was thinking.